### ISSUE TITLE
ledger api server: bump ledger api version to 2.0 and update docs

### ISSUE THREAD

**Issue by adriaanm-da** [digital-asset/daml#12174]
must include:
- [ ] check what other past commits to https://docs.daml.com/support/compatibility.html changed, and do these changes as well
- [ ] adapt https://docs.daml.com/support/compatibility.html
- [ ] search for other docs that talk about version and adapt them
- [ ] change ledger api server to return the right version
- [ ] manually test that the 2.0 version is reported

**Comment by adriaanm-da** [comment]
Depends on #12173

**Comment by cocreature** [comment]
other way around, #12173 depends on this.

**Comment by mziolekda** [comment]
@adriaanm-da Why do we need this:
 adapt test tool to check that if user management is supported then ledger api version is > 2.0

We do not run a risk that 1.18 will spontaneously implement user management

**Comment by adriaanm-da** [comment]
:-) I don't know, I think @meiersi-da wrote that.

**Comment by meiersi-da** [comment]
@mziolekda : yeah, scrap that. That was a brainfart. I was looking for some test that ensured 2.0 is reported. A manual test suffices though.

**Comment by mziolekda** [comment]
Closing, obsolete issue
